src/components/Button.js

```diff
@@ -1,16 +1,38 @@
 import PropTypes from 'prop-types';
 
 const Button = (props) => {
-  const { value } = props;
-  return <button className="button" type="button">{value}</button>;
+  const { value, bgColor, width } = props;
+  let className = 'button';
+
+  if (bgColor === 'orange') {
+    className += ' button-orange';
+  } else {
+    className += ' button-lightGray';
+  }
+
+  if (width) {
+    className += ' button-big';
+  } else {
+    className += ' button-small';
+  }
+
+  return (
+    <button className={className} type="button">
+      {value}
+    </button>
+  );
 };
 
 Button.propTypes = {
   value: PropTypes.string,
+  bgColor: PropTypes.string,
+  width: PropTypes.number,
 };
 
 Button.defaultProps = {
   value: '',
+  bgColor: '',
```

Comment:
- [ ] [ REQUIRED] default `color` for `button` is `orange` ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/react_reducx/calculator/milestone3/colorProps.png)). Here, the default value of `bgColor` is `an empty string`

Reply:
Default color is light-gray and it is implemented now.

Reply:
I've fixed the "propType" error too.

Reply:
> Default color is light-gray and it is implemented now.

The default color should be orange